parallel_execute_scripts/run_execute.sh

```diff
@@ -0,0 +1 @@
+nohup sudo ssh -i /opt/yugabyte/data/keys/08c0ba0e-3558-40fc-94e5-a87a627de8c5/yb-15-aws-portal-1-key.pem -p 54422 -ostricthostkeychecking=no centos@$1 'sudo chmod +x ~/execute.sh; cd tpcc; ../execute.sh' > /tmp/$1_execute.txt &
```

Comment:
This `opt/yugabyte/data/keys/08c0ba0e-3558-40fc-94e5-a87a627de8c5/yb-15-aws-portal-1-key.pem` path will not always be the same right?
Can we make some of these stuffs for the run_execute, run_loader, and upload_scripts parametric, or maybe we can generate them too.

Reply:
Done.